docs/dependency-injection/index.md

```diff
@@ -11,7 +11,7 @@ As a result you can easily implement support for a container that is not otherwi
 
 ## Next Steps
 
-- Learn how to [Register Services](registering-types.md) and [Register Pages for Navigation](../navigation/navigation-basics.md)
```

Comment:
it would appear that the correct link would be: ../xamarin-forms/navigation/navigation-basics.md